Gemfile

```diff
@@ -10,8 +10,9 @@ end
 gem 'deprecation'
 gem 'honeybadger'
 gem 'puma', '~> 5.3' # the app server
-gem 'rails', '~> 5.2.0'
-gem 'roo', '>= 2.7.1'
+gem 'rails', '~> 6.1.0'
+# Pin roo dependency to a thus far unreleased commit that brings Ruby 3 compatibility
+gem 'roo', github: 'roo-rb/roo', ref: '868d4ea419cf393c9d8832838d96c82e47116d2f' # gem 'roo', '>= 2.7.1'
```

Comment:
ew - are we really using roo???   why?

Reply:
I could prolly `git blame` it but 🤷🏻‍♂️ ... yeah

Reply:
To parse spreadsheet uploads